src/components/controls/PercentileSelectionControl.svelte

```diff
@@ -1,36 +1,17 @@
 <script>
-import { percentileLineColorMap as cmp } from 'udgl/data-graphics/utils/color-maps';
-import BodyControl from './BodyControl.svelte';
+  import { percentileLineColorMap as cmp } from '../../utils/color-maps';
+  import BodyControl from './BodyControl.svelte';
 
-export let percentiles = [50];
-
-function t(v) {
-  // We can even return a string built using a template literal
-  return `${v}% of clients have seen this value or one smaller`;
-}
+  export let percentiles = [50];
 
+  function t(v) {
+    // We can even return a string built using a template literal
+    return `${v}% of clients have seen this value or one smaller`;
+  }
 </script>
-<BodyControl
-  options={[
-      {
-        label: '5%', value: 5, labelColor: cmp(5), tooltip: t(5),
-      },
-      {
-        label: '25%', value: 25, labelColor: cmp(25), tooltip: t(25),
-      },
-      {
-        label: '50%', value: 50, labelColor: cmp(50), tooltip: t(50),
-      },
-      {
-        label: '75%', value: 75, labelColor: cmp(75), tooltip: t(75),
-      },
-      {
-        label: '95%', value: 95, labelColor: cmp(95), tooltip: t(95),
-      },
-    ]}
-    reverse={true}
-    bind:selected={percentiles}
-    multi={true}
-    level="low"
-    on:selection
-/>
\ No newline at end of file
+<BodyControl options={[{ label: '5%', value: 5, labelColor: cmp(5), tooltip:
+t(5), }, { label: '25%', value: 25, labelColor: cmp(25), tooltip: t(25), }, {
+label: '50%', value: 50, labelColor: cmp(50), tooltip: t(50), }, { label: '75%',
+value: 75, labelColor: cmp(75), tooltip: t(75), }, { label: '95%', value: 95,
+labelColor: cmp(95), tooltip: t(95), }]} reverse={true}
```

Comment:
```
  16:22  error  Object properties must go on a new line if they aren't all on the same line  object-property-newline
  16:36  error  Unexpected trailing comma                                                    comma-dangle
  16:38  error  Expected a line break before this closing brace                              object-curly-newline
```

I know a lot of these ESLint problems may not be new, but if we can fix them while we're modifying and testing these files, that may be easier than fixing them down the road,

Reply:
not goinig to comment on all of these, but it seems like something is pretty off w/ our svelte-prettier configuration here if it formats things in a way that violates eslint. You can see here that this issue _is_ actually a new eslint error, caused by prettier (I'm guessing). Does your editor format it properly?